mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerBaseTest.java

```diff
@@ -230,6 +276,250 @@ public void testJoinTables(FileFormat format) throws IOException {
     Assert.assertArrayEquals(new Object[] {1L, "Bob", 102L, 33.33d}, rows.get(2));
   }
 
+  private void testArrayTypeInTable(FileFormat format) throws IOException {
+    final Schema schema = new Schema(
+            required(1, "arrayofprimitives", Types.ListType.ofRequired(
```

Comment:
Style:
* We don't use `final` for local variables because Java does a good job of inferring it and it is debatable whether it actually helps the compiler make better decisions.
* The indentation is off. It should be 4 spaces (2 indents) for a continuing indent.